ui/admin/tests/helpers/selectors.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+export const SAVE_BTN = '[type=submit]';
+export const CANCEL_BTN = '.rose-form-actions [type=button]';
+export const EDIT_BTN = '.rose-form-actions [type=button]';
+
+export const FIELD_NAME = '[name=name]';
+export const FIELD_NAME_VALUE = 'Random name';
+
+export const TABLE_ROW_RESOURCE_LINK = 'tbody tr:first-child a';
```

Comment:
Should this include something in the name that mentions it's the first row?

Reply:
yeah that would be more clear. I will update that name.